rasa/core/channels/hangouts.py

```diff
@@ -0,0 +1,288 @@
+import asyncio
+import inspect
+import json
+import logging
+import uuid
+from asyncio import Queue, CancelledError
+from sanic import Sanic, Blueprint, response
+from sanic.request import Request
+from typing import Text, List, Dict, Any, Optional, Callable, Iterable, Awaitable, Union
+
+import rasa.utils.endpoints
+from rasa.cli import utils as cli_utils
+from rasa.constants import DOCS_BASE_URL
+from rasa.core import utils
+from sanic.response import HTTPResponse
+
+from rasa.core.channels.channel import InputChannel, OutputChannel, UserMessage
+
+try:
+    from urlparse import urljoin  # pytype: disable=import-error
+except ImportError:
+    from urllib.parse import urljoin
+
+logger = logging.getLogger(__name__)
+
+
+class HangoutsOutput(OutputChannel):
+    @classmethod
+    def name(cls) -> Text:
+        return "hangouts"
+
+    def __init__(self) -> None:
+        self.messages = {}
+
+    def _text_card(self, message: Dict[Text, Any]) -> Dict:
+
+        card = {
+            "cards": [
+                {
+                    "sections": [
+                        {"widgets": [{"textParagraph": {"text": message["text"]}}]}
+                    ]
+                }
+            ]
+        }
+        return card
+
+    def _image_card(self, image: Text) -> Dict:
+        card = {
+            "cards": [{"sections": [{"widgets": [{"image": {"imageUrl": image}}]}]}]
+        }
+        return card
+
+    def _text_button_card(self, text: Text, buttons: List) -> Dict:
+        hangouts_buttons = []
+        for b in buttons:
+            try:
+                b_txt, b_pl = b["title"], b["payload"]
+            except KeyError:
+                logger.error(
+                    "Buttons must be a list of dicts with 'title' and 'payload' as keys"
+                )
```

Comment:
does it make sense to whether `pass` or `return` after Exception?

Reply:
Added return